### PR TITLE
cloud: add doc for Chat2Query

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/connect-via-chat2query.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+TiDB Cloud is powered by AI. You can use Chat2Query (beta), an AI-powered SQL editor in the [TiDB Cloud console](https://tidbcloud.com/), to maximize your data value.
+
+In Chat2Query, you can either simply type `--` and your instructions to let AI generate SQL queries automatically or write SQL queries manually, and then run SQL queries against databases without a terminal. You can find the query results in tables intuitively and check the query logs easily.
+
+> **Note:**
+>
+> - Chat2Query is only available to [Serverless Tier](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta) clusters.
+> - The AI capacity of Chat2Query relies on data to work. It collects data to provide the service, some of which is then retained for further product improvements. After you enable the AI capacity, Chat2Query only collects your table schema information and will not collect any specific data. For more information, see [Chat2Query Privacy FAQ](https://www.pingcap.com/privacy-policy/privacy-chat2query).
+
+## Limitation
```
Comment (qiancai) [inline tidb-cloud/connect-via-chat2query.md:17]:
```suggestion
## Limitations
```

File: tidb-cloud/tidb-cloud-quickstart.md [hunk 2]
```diff
@@ -31,26 +31,34 @@ TiDB Cloud [Serverless Tier](/tidb-cloud/select-cluster-tier.md#serverless-tier-
 
     The cluster creation process starts and your Serverless Tier cluster will be created in approximately 30 seconds.
 
-## Step 2. Try Chat2Query (Beta)
+## Step 2. Try TiDB Cloud AI capacity
 
-Chat2Query is an AI-powered SQL editor that allows you to directly edit and run SQL queries against databases of Serverless Tier without using a terminal. In Chat2Query, you can use AI to generate SQL queries automatically.
+TiDB Cloud is powered by AI. You can use Chat2Query (beta), an AI-powered SQL editor in the TiDB Cloud console, to maximize your data value.
+
+ In Chat2Query, you can either simply type `--` and your instructions to let AI generate SQL queries automatically or write SQL queries manually, and then run SQL queries against databases without a terminal.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-quickstart.md:38]:
```suggestion
In Chat2Query, you can either simply type `--` and your instructions to let AI generate SQL queries automatically or write SQL queries manually, and then run SQL queries against databases without a terminal.
```

File: tidb-cloud/explore-data-with-chat2query.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+        For a SQL query generated by AI, you can accept it by pressing `Tab` and then further edit it if needed, or reject it by pressing `Esc`.
+
+    - If AI is disabled, write SQL queries manually.
+
+2. Run SQL queries.
+
+    <SimpleTab>
+    <div label="Mac">
+
+    For Mac:
```
Comment (lilin90) [inline tidb-cloud/explore-data-with-chat2query.md:69]:
```suggestion
    For macOS:
```

File: tidb-cloud/explore-data-with-chat2query.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+
+In Chat2Query, you can either simply type `--` and your instructions to let AI generate SQL queries automatically or write SQL queries manually, and then run SQL queries against databases without a terminal. You can find the query results in tables intuitively and check the query logs easily.
+
+> **Note:**
+>
+> Chat2Query is only available to [Serverless Tier](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta) clusters.
+
+## Usage scenarios
+
+The recommended usage scenarios of Chat2Query are as follows:
```
Comment (qiancai) [inline tidb-cloud/explore-data-with-chat2query.md:18]:
```suggestion
## Use cases

The recommended use cases of Chat2Query are as follows:
```

File: tidb-cloud/tidb-cloud-quickstart.md [hunk 5]
```diff
@@ -31,26 +31,51 @@ TiDB Cloud [Serverless Tier](/tidb-cloud/select-cluster-tier.md#serverless-tier-
...
     >
-    > Currently, Chat2Query has limited support on SQL statements. DDLs such as `CREATE TABLE` or `DROP TABLE` are not supported yet.
+    > SQL queries generated by AI is not 100% accurate and might still need your further tweak.
+
+4. Run SQL queries.
+
+   <SimpleTab>
+    <div label="Mac">
+
+    For Mac:
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-quickstart.md:55]:
```suggestion
    For macOS:
```

File: tidb-cloud/explore-data-with-chat2query.md [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+
+> **Note:**
+>
+> Chat2Query is only available in [Serverless Tier](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta) clusters.
+
+## Use cases
+
+The recommended use cases of Chat2Query are as follows:
+
+- Use the AI capacity of Chat2Query to help you write complex SQL queries instantly.
```
Comment (qiancai) [inline tidb-cloud/explore-data-with-chat2query.md:20]:
```suggestion
- Use the AI capacity of Chat2Query to help you generate complex SQL queries instantly.
```

File: tidb-cloud/connect-to-tidb-cluster.md [hunk 7]
```diff
@@ -17,9 +17,11 @@ For Serverless Tier clusters, you can connect to your cluster via standard conne
 
     Serverless Tier only [supports TLS connections](/tidb-cloud/secure-connections-to-serverless-tier-clusters.md), which ensures the security of data transmission from your applications to TiDB clusters.
 
-- Chat2Query (Beta)
+- [Chat2Query (Beta)](/tidb-cloud/explore-data-with-chat2query.md)
```
Comment (qiancai) [inline tidb-cloud/connect-to-tidb-cluster.md:20]:
```suggestion
- [Chat2Query (beta)](/tidb-cloud/explore-data-with-chat2query.md)
```

File: tidb-cloud/connect-to-tidb-cluster.md [hunk 8]
```diff
@@ -17,9 +17,11 @@ For Serverless Tier clusters, you can connect to your cluster via standard conne
 
     Serverless Tier only [supports TLS connections](/tidb-cloud/secure-connections-to-serverless-tier-clusters.md), which ensures the security of data transmission from your applications to TiDB clusters.
 
-- Chat2Query (Beta)
+- [Chat2Query (beta)](/tidb-cloud/explore-data-with-chat2query.md)
 
-    Chat2Query is an AI-powered SQL editor in the [TiDB Cloud console](https://tidbcloud.com/), which allows you to directly edit and run SQL queries against databases of Serverless Tier. In Chat2Query, you can type `--` and your instruction to let AI generate SQL queries automatically. To access Chat2Query, go to the [**Clusters**](https://tidbcloud.com/console/clusters) page, choose a target project in the left navigation pane if you have multiple projects, click your cluster name, and then click **Chat2Query** in the left navigation pane.
+    TiDB Cloud is powered by AI. You can use Chat2Query (beta), an AI-powered SQL editor in the [TiDB Cloud console](https://tidbcloud.com/), to maximize your data value.
```
Comment (Oreoxmt) [inline tidb-cloud/connect-to-tidb-cluster.md:22]:
```suggestion
    TiDB Cloud is powered by artificial intelligence (AI). You can use Chat2Query (beta), an AI-powered SQL editor in the [TiDB Cloud console](https://tidbcloud.com/), to maximize your data value.
```

File: tidb-cloud/connect-to-tidb-cluster.md [hunk 9]
```diff
@@ -17,9 +17,11 @@ For Serverless Tier clusters, you can connect to your cluster via standard conne
...
+
+    In Chat2Query, you can either simply type `--` and your instructions to let AI generate SQL queries automatically or write SQL queries manually, and then run SQL queries against databases without a terminal. You can find the query results in tables intuitively and check the query logs easily.
```
Comment (Oreoxmt) [inline tidb-cloud/connect-to-tidb-cluster.md:24]:
```suggestion
    In Chat2Query, you can either simply type `--` followed by your instructions to let AI generate SQL queries automatically or write SQL queries manually, and then run SQL queries against databases without a terminal. You can find the query results in tables intuitively and check the query logs easily.
```

File: tidb-cloud/tidb-cloud-quickstart.md [hunk 10]
```diff
@@ -31,26 +31,51 @@ TiDB Cloud [Serverless Tier](/tidb-cloud/select-cluster-tier.md#serverless-tier-
...
 1. On the [**Clusters**](https://tidbcloud.com/console/clusters) page, click your cluster name, and then click **Chat2Query** in the left navigation pane.
 
+2. To try TiDB Cloud AI capacity, follow the on-screen instructions to allow PingCAP and OpenAI to use your code snippets to research and improve the services, and then click **Save and Get Started**.
+
+3. In the editor, you can either simply type `--` and your instructions to let AI generate SQL queries automatically or write SQL queries manually.
+
     > **Note:**
     >
-    > Currently, Chat2Query has limited support on SQL statements. DDLs such as `CREATE TABLE` or `DROP TABLE` are not supported yet.
+    > SQL queries generated by AI is not 100% accurate and might still need your further tweak.
```
Comment (Oreoxmt) [inline tidb-cloud/tidb-cloud-quickstart.md:48]:
```suggestion
    > SQL queries generated by AI are not 100% accurate and might still need your further tweak.
```

File: tidb-cloud/explore-data-with-chat2query.md [hunk 11]
```diff
@@ -0,0 +1,110 @@
...
+
+    > **Tip:**
+    >
+    > If you have multiple projects, you can switch to the target project in the left navigation pane of the **Clusters** page.
+
+2. Click your cluster name, and then click **Chat2Query** in the left navigation pane.
+
+## Enable or disable AI to generate SQL queries
+
+PingCAP takes the privacy and security of users' data as top priority. The AI capacity of Chat2Query only needs to access database schemas to generate SQL queries, not your data itself. For more information, see [Chat2Query Privacy FAQ](https://www.pingcap.com/privacy-policy/privacy-chat2query).
```
Comment (Oreoxmt) [inline tidb-cloud/explore-data-with-chat2query.md:40]:
```suggestion
PingCAP takes the privacy and security of users' data as a top priority. The AI capacity of Chat2Query only needs to access database schemas to generate SQL queries, not your data itself. For more information, see [Chat2Query Privacy FAQ](https://www.pingcap.com/privacy-policy/privacy-chat2query).
```

File: tidb-cloud/explore-data-with-chat2query.md [hunk 12]
```diff
@@ -0,0 +1,110 @@
...
+
+The recommended use cases of Chat2Query are as follows:
+
+- Use the AI capacity of Chat2Query to help you generate complex SQL queries instantly.
+- Test out the MySQL compatibility of TiDB quickly.
+- Explore TiDB SQL features easily.
+
+## Limitation
+
+SQL queries generated by AI is not 100% accurate and might still need your further tweak.
```
Comment (Oreoxmt) [inline tidb-cloud/explore-data-with-chat2query.md:26]:
```suggestion
SQL queries generated by AI are not 100% accurate and might still need your further tweak.
```

File: tidb-cloud/explore-data-with-chat2query.md [hunk 13]
```diff
@@ -0,0 +1,110 @@
...
+    >
+    > If you have multiple projects, you can switch to the target project in the left navigation pane of the **Clusters** page.
+
+2. Click your cluster name, and then click **Chat2Query** in the left navigation pane.
+
+## Enable or disable AI to generate SQL queries
+
+PingCAP takes the privacy and security of users' data as top priority. The AI capacity of Chat2Query only needs to access database schemas to generate SQL queries, not your data itself. For more information, see [Chat2Query Privacy FAQ](https://www.pingcap.com/privacy-policy/privacy-chat2query).
+
+For the first-time you access Chat2Query, you will get a dialog about whether to allow PingCAP and OpenAI to use your code snippets to research and improve the services.
```
Comment (Oreoxmt) [inline tidb-cloud/explore-data-with-chat2query.md:42]:
```suggestion
When you access Chat2Query for the first time, you will see a dialog about whether to allow PingCAP and OpenAI to use your code snippets to research and improve the services.
```

File: tidb-cloud/tidb-cloud-quickstart.md [hunk 14]
```diff
@@ -31,26 +31,51 @@ TiDB Cloud [Serverless Tier](/tidb-cloud/select-cluster-tier.md#serverless-tier-
...
+
+4. Run SQL queries.
+
+   <SimpleTab>
+    <div label="Mac">
```
Comment (Oreoxmt) [inline tidb-cloud/tidb-cloud-quickstart.md:53]:
```suggestion
    <div label="macOS">
```

File: tidb-cloud/explore-data-with-chat2query.md [hunk 15]
```diff
@@ -0,0 +1,110 @@
...
+    - If AI is disabled, write SQL queries manually.
+
+2. Run SQL queries.
+
+    <SimpleTab>
+    <div label="Mac">
+
+    For macOS:
+
+    - If you have only one query in the editor, to run it, press **Command + Enter** or click <svg width="1rem" height="1rem" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M6.70001 20.7756C6.01949 20.3926 6.00029 19.5259 6.00034 19.0422L6.00034 12.1205L6 5.33028C6 4.75247 6.00052 3.92317 6.38613 3.44138C6.83044 2.88625 7.62614 2.98501 7.95335 3.05489C8.05144 3.07584 8.14194 3.12086 8.22438 3.17798L19.2865 10.8426C19.2955 10.8489 19.304 10.8549 19.3126 10.8617C19.4069 10.9362 20 11.4314 20 12.1205C20 12.7913 19.438 13.2784 19.3212 13.3725C19.307 13.3839 19.2983 13.3902 19.2831 13.4002C18.8096 13.7133 8.57995 20.4771 8.10002 20.7756C7.60871 21.0812 7.22013 21.0683 6.70001 20.7756Z" fill="currentColor"></path></svg>**Run**.
```
Comment (Oreoxmt) [inline tidb-cloud/explore-data-with-chat2query.md:71]: On macOS, I can use either **Control + Enter** or **Command + Enter** to run SQL.

Comment (qiancai) [reply tidb-cloud/explore-data-with-chat2query.md:71]: According to PM, this is to provide the same instructions as that on the UI.

File: tidb-cloud/explore-data-with-chat2query.md [hunk 16]
```diff
@@ -0,0 +1,110 @@
+---
+title: Explore Your Data with AI-Powered Chat2Query (beta)
+summary: Learn how to use Chat2Query, an AI-powered SQL editor, to maximize your data value.
+---
+
+# Explore Your Data with AI-Powered Chat2Query (beta)
+
+TiDB Cloud is powered by AI. You can use Chat2Query (beta), an AI-powered SQL editor in the [TiDB Cloud console](https://tidbcloud.com/), to maximize your data value.
+
+In Chat2Query, you can either simply type `--` and your instructions to let AI generate SQL queries automatically or write SQL queries manually, and then run SQL queries against databases without a terminal. You can find the query results in tables intuitively and check the query logs easily.
```
Comment (qiancai) [inline tidb-cloud/explore-data-with-chat2query.md:10]:
```suggestion
In Chat2Query, you can either simply type `--` followed by your instructions to let AI generate SQL queries automatically or write SQL queries manually, and then run SQL queries against databases without a terminal. You can find the query results in tables intuitively and check the query logs easily.
```

File: tidb-cloud/explore-data-with-chat2query.md [hunk 17]
```diff
@@ -0,0 +1,110 @@
...
+- To enable AI, click **Enable AI power for data exploration** in the upper-right corner of Chat2Query.
+- To disable AI, click <MDSvgIcon name="icon-top-account-settings" /> **Account** in the upper-right corner of the [TiDB Cloud console](https://tidbcloud.com/), click **Account Settings**, click the **Privacy** tab, and then disable the **AI-powered Data Exploration** option.
+
+## Write and run SQL queries
+
+In Chat2Query, you can write and run SQL queries using the pre-built sample dataset or your own dataset.
+
+1. Write SQL queries.
+
+    - If AI is enabled, simply type `--` and your instructions to let AI generate SQL queries automatically or write SQL queries manually.
```
Comment (qiancai) [inline tidb-cloud/explore-data-with-chat2query.md:58]:
```suggestion
    - If AI is enabled, simply type `--` followed by your instructions to let AI generate SQL queries automatically or write SQL queries manually.
```

File: tidb-cloud/tidb-cloud-quickstart.md [hunk 18]
```diff
@@ -31,26 +31,51 @@ TiDB Cloud [Serverless Tier](/tidb-cloud/select-cluster-tier.md#serverless-tier-
 
     The cluster creation process starts and your Serverless Tier cluster will be created in approximately 30 seconds.
 
-## Step 2. Try Chat2Query (Beta)
+## Step 2. Try AI-powered Chat2Query
 
-Chat2Query is an AI-powered SQL editor that allows you to directly edit and run SQL queries against databases of Serverless Tier without using a terminal. In Chat2Query, you can use AI to generate SQL queries automatically.
+TiDB Cloud is powered by AI. You can use Chat2Query (beta), an AI-powered SQL editor in the TiDB Cloud console, to maximize your data value.
+
+In Chat2Query, you can either simply type `--` and your instructions to let AI generate SQL queries automatically or write SQL queries manually, and then run SQL queries against databases without a terminal.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-quickstart.md:38]:
```suggestion
In Chat2Query, you can either simply type `--` followed by your instructions to let AI generate SQL queries automatically or write SQL queries manually, and then run SQL queries against databases without a terminal.
```

File: tidb-cloud/tidb-cloud-quickstart.md [hunk 19]
```diff
@@ -31,26 +31,51 @@ TiDB Cloud [Serverless Tier](/tidb-cloud/select-cluster-tier.md#serverless-tier-
...
 
 1. On the [**Clusters**](https://tidbcloud.com/console/clusters) page, click your cluster name, and then click **Chat2Query** in the left navigation pane.
 
+2. To try TiDB Cloud AI capacity, follow the on-screen instructions to allow PingCAP and OpenAI to use your code snippets to research and improve the services, and then click **Save and Get Started**.
+
+3. In the editor, you can either simply type `--` and your instructions to let AI generate SQL queries automatically or write SQL queries manually.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-quickstart.md:44]:
```suggestion
3. In the editor, you can either simply type `--` followed by your instructions to let AI generate SQL queries automatically or write SQL queries manually.
```

**Review by zhoubasten — APPROVED** [review]
LGTM

**Review by sanshuiyijing — APPROVED** [review]
LGTM

File: tidb-cloud/explore-data-with-chat2query.md [hunk 20]
```diff
@@ -0,0 +1,110 @@
...
+
+    - If you have multiple queries in the editor, to run one or several of them sequentially, select the lines of the target queries with your cursor, and then press **Ctrl + Enter** or click **Run**.
+
+    - To run all queries in the editor sequentially, press **Shift + Ctrl + Enter**, or select the lines of all queries with your cursor and click **Run**.
+
+    </div>
+    </SimpleTab>
+
+After running the queries, you can see the query log and results immediately at the bottom of the page.
```
Comment (Oreoxmt) [inline tidb-cloud/explore-data-with-chat2query.md:92]:
```suggestion
After running the queries, you can see the query logs and results immediately at the bottom of the page.
```

File: tidb-cloud/tidb-cloud-quickstart.md [hunk 21]
```diff
@@ -31,26 +31,51 @@ TiDB Cloud [Serverless Tier](/tidb-cloud/select-cluster-tier.md#serverless-tier-
...
-    - To run two queries sequentially, you can do one of the following:
+    - To run all queries in the editor sequentially, press **Shift + Ctrl + Enter**, or select the lines of all queries with your cursor and click **Run**.
 
-        * Press **Control + Shift + Enter**.
-        * Select the two queries with your cursor and click **Run**.
+    </div>
+    </SimpleTab>
 
-    - If there are multiple queries in the editor and you only want to run one of them, select the target query with your cursor, press **Control + Enter** or click **Run**.
+After running the queries, you can see the query log and results immediately at the bottom of the page.
```
Comment (Oreoxmt) [inline tidb-cloud/tidb-cloud-quickstart.md:78]:
```suggestion
After running the queries, you can see the query logs and results immediately at the bottom of the page.
```

File: tidb-cloud/explore-data-with-chat2query.md [hunk 22]
```diff
@@ -0,0 +1,110 @@
...
+
+## Manage SQL files
+
+In Chat2Query, you can save your SQL queries in different SQL files and manage SQL files as follows:
+
+- To add a SQL file, click **+** on the **SQL Files** tab.
+- To rename a SQL file, move your cursor on the filename, click **...** next to the filename, and then select **Rename**.
```
Comment (Oreoxmt) [inline tidb-cloud/explore-data-with-chat2query.md:99]:
```suggestion
- To rename an existing SQL file, move your cursor on the filename, click **...** next to the filename, and then select **Rename**.
```

File: tidb-cloud/explore-data-with-chat2query.md [hunk 23]
```diff
@@ -0,0 +1,110 @@
...
+- To delete a SQL file, move your cursor on the filename, click **...** next to the filename, and then select **Delete**.
```
Comment (Oreoxmt) [inline tidb-cloud/explore-data-with-chat2query.md:100]:
```suggestion
- To delete an existing SQL file, move your cursor on the filename, click **...** next to the filename, and then select **Delete**.
```

File: tidb-cloud/explore-data-with-chat2query.md [hunk 24]
```diff
@@ -0,0 +1,110 @@
...
+
+## Manage Chat2Query settings
+
+By default, Chat2Query limits the maximum number of rows in query results to 500 and does not show system database schemas on the **Schemas** tab.
+
+To change the settings, take the following steps:
+
+1. In the upper-right corner of Chat2Query, click **...** and select **Chat2Query settings**.
```
Comment (Oreoxmt) [inline tidb-cloud/explore-data-with-chat2query.md:108]:
```suggestion
1. In the upper-right corner of Chat2Query, click **...** and select **Settings**.
```
image

File: tidb-cloud/tidb-cloud-quickstart.md [hunk 25]
```diff
@@ -31,26 +31,51 @@ TiDB Cloud [Serverless Tier](/tidb-cloud/select-cluster-tier.md#serverless-tier-
...
 
-## Step 2. Try Chat2Query (Beta)
+## Step 2. Try AI-powered Chat2Query
 
-Chat2Query is an AI-powered SQL editor that allows you to directly edit and run SQL queries against databases of Serverless Tier without using a terminal. In Chat2Query, you can use AI to generate SQL queries automatically.
+TiDB Cloud is powered by AI. You can use Chat2Query (beta), an AI-powered SQL editor in the TiDB Cloud console, to maximize your data value.
+
+In Chat2Query, you can either simply type `--` followed by your instructions to let AI generate SQL queries automatically or write SQL queries manually, and then run SQL queries against databases without a terminal.
 
 1. On the [**Clusters**](https://tidbcloud.com/console/clusters) page, click your cluster name, and then click **Chat2Query** in the left navigation pane.
```
Comment (Oreoxmt) [inline tidb-cloud/tidb-cloud-quickstart.md:40]:
```suggestion
1. On the [**Clusters**](https://tidbcloud.com/console/clusters) page, click your cluster name, and then click <MDSvgIcon name="icon-left-sql-editor" /> **Chat2Query** in the left navigation pane.
```

File: tidb-cloud/explore-data-with-chat2query.md [hunk 26]
```diff
@@ -0,0 +1,110 @@
...
+
+    - To run all queries in the editor sequentially, press **Shift + Command + Enter**, or select the lines of all queries with your cursor and click **Run**.
+
+    </div>
+
+    <div label="Windows/Linux">
+
+    For Windows or Linux:
+
+    - If you have only one query in the editor, to run it, press **Ctrl + Enter** or click <svg width="1rem" height="1rem" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M6.70001 20.7756C6.01949 20.3926 6.00029 19.5259 6.00034 19.0422L6.00034 12.1205L6 5.33028C6 4.75247 6.00052 3.92317 6.38613 3.44138C6.83044 2.88625 7.62614 2.98501 7.95335 3.05489C8.05144 3.07584 8.14194 3.12086 8.22438 3.17798L19.2865 10.8426C19.2955 10.8489 19.304 10.8549 19.3126 10.8617C19.4069 10.9362 20 11.4314 20 12.1205C20 12.7913 19.438 13.2784 19.3212 13.3725C19.307 13.3839 19.2983 13.3902 19.2831 13.4002C18.8096 13.7133 8.57995 20.4771 8.10002 20.7756C7.60871 21.0812 7.22013 21.0683 6.70001 20.7756Z" fill="currentColor"></path></svg>**Run**.
```
[the remark this answers is not in the document]
Comment (qiancai) [reply tidb-cloud/explore-data-with-chat2query.md:83]: This is to align with UI. 
img_v2_fe0f2ebb-8bab-4a5f-985e-a8b53d2baecg

File: tidb-cloud/explore-data-with-chat2query.md [hunk 27]
```diff
@@ -0,0 +1,110 @@
...
+
+    - If you have multiple queries in the editor, to run one or several of them sequentially, select the lines of the target queries with your cursor, and then press **Ctrl + Enter** or click **Run**.
```
[the remark this answers is not in the document]
Comment (qiancai) [reply tidb-cloud/explore-data-with-chat2query.md:85]: ditto

File: tidb-cloud/explore-data-with-chat2query.md [hunk 28]
```diff
@@ -0,0 +1,110 @@
...
+
+    - To run all queries in the editor sequentially, press **Shift + Ctrl + Enter**, or select the lines of all queries with your cursor and click **Run**.
```
[the remark this answers is not in the document]
Comment (qiancai) [reply tidb-cloud/explore-data-with-chat2query.md:87]: ditto

File: tidb-cloud/explore-data-with-chat2query.md (hunk 22 above)
Comment (qiancai) [reply tidb-cloud/explore-data-with-chat2query.md:99]: not necessary? It seems that no one will rename a file on the GUI that is not existing.

File: tidb-cloud/explore-data-with-chat2query.md (hunk 23 above)
Comment (qiancai) [reply tidb-cloud/explore-data-with-chat2query.md:100]: not necessary? It seems that no one will delete a file on the GUI that is not existing.

File: tidb-cloud/explore-data-with-chat2query.md (hunk 24 above)
Comment (qiancai) [reply tidb-cloud/explore-data-with-chat2query.md]: Good catch. It was newly updated today.

File: tidb-cloud/explore-data-with-chat2query.md [hunk 29]
```diff
@@ -0,0 +1,110 @@
+---
+title: Explore Your Data with AI-Powered Chat2Query (beta)
+summary: Learn how to use Chat2Query, an AI-powered SQL editor, to maximize your data value.
```
Comment (Oreoxmt) [inline tidb-cloud/explore-data-with-chat2query.md:3]:
```suggestion
summary: Learn how to use Chat2Query, an AI-powered SQL editor in the TiDB Cloud console, to maximize your data value.
```

File: tidb-cloud/tidb-cloud-quickstart.md [hunk 30]
```diff
@@ -82,6 +107,7 @@ The data import process will take several minutes. When the data import progress
 
 ## What's next
 
-- For how to connect to your cluster via a SQL client, refer to [Connect to a TiDB Cluster](/tidb-cloud/connect-to-tidb-cluster.md).
-- For more details on TiDB SQL usage, see [Explore SQL with TiDB](/basic-sql-operations.md).
-- For production use with the benefits of cross-zone high availability, horizontal scaling, and [HTAP](https://en.wikipedia.org/wiki/Hybrid_transactional/analytical_processing), refer to [Create a TiDB Cluster](/tidb-cloud/create-tidb-cluster.md) and create a Dedicated Tier cluster.
+- For how to connect to your cluster via a SQL client, see [Connect to a TiDB Cluster](/tidb-cloud/connect-to-tidb-cluster.md).
```
Comment (Oreoxmt) [inline tidb-cloud/tidb-cloud-quickstart.md:110]:
```suggestion
- For how to connect to your cluster via different methods, see [Connect to a TiDB cluster](/tidb-cloud/connect-to-tidb-cluster.md).
```

File: tidb-cloud/tidb-cloud-quickstart.md [hunk 31]
```diff
@@ -82,6 +107,7 @@ The data import process will take several minutes. When the data import progress
...
+- For more information about how to use Chat2Query to explore your data, see [Chat2Query](/tidb-cloud/explore-data-with-chat2query.md).
+- For TiDB SQL usage, see [Explore SQL with TiDB](/basic-sql-operations.md).
+- For production use with the benefits of cross-zone high availability, horizontal scaling, and [HTAP](https://en.wikipedia.org/wiki/Hybrid_transactional/analytical_processing), see [Create a TiDB Cluster](/tidb-cloud/create-tidb-cluster.md) to create a Dedicated Tier cluster.
```
Comment (Oreoxmt) [inline tidb-cloud/tidb-cloud-quickstart.md:113]:
```suggestion
- For production use with the benefits of cross-zone high availability, horizontal scaling, and [HTAP](https://en.wikipedia.org/wiki/Hybrid_transactional/analytical_processing), see [Create a TiDB cluster](/tidb-cloud/create-tidb-cluster.md) to create a Dedicated Tier cluster.
```

File: tidb-cloud/explore-data-with-chat2query.md [hunk 32]
```diff
@@ -0,0 +1,110 @@
...
+
+After running the queries, you can see the query logs and results immediately at the bottom of the page.
+
+## Manage SQL files
+
+In Chat2Query, you can save your SQL queries in different SQL files and manage SQL files as follows:
+
+- To add a SQL file, click **+** on the **SQL Files** tab.
+- To rename a SQL file, move your cursor on the filename, click **...** next to the filename, and then select **Rename**.
+- To delete a SQL file, move your cursor on the filename, click **...** next to the filename, and then select **Delete**.
```
Comment (qiancai) [inline tidb-cloud/explore-data-with-chat2query.md:100]:
```suggestion
- To delete a SQL file, move your cursor on the filename, click **...** next to the filename, and then select **Delete**. Note that when there is only one SQL file on the **SQL Files** tab, you cannot delete it.
```

File: tidb-cloud/tidb-cloud-quickstart.md [hunk 33]
```diff
@@ -31,26 +31,51 @@ TiDB Cloud [Serverless Tier](/tidb-cloud/select-cluster-tier.md#serverless-tier-
 
     The cluster creation process starts and your Serverless Tier cluster will be created in approximately 30 seconds.
 
-## Step 2. Try Chat2Query (Beta)
+## Step 2. Try AI-powered Chat2Query
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-quickstart.md:34]:
```suggestion
## Step 2. Try AI-powered Chat2Query (beta)
```

File: tidb-cloud/explore-data-with-chat2query.md [hunk 34]
```diff
@@ -0,0 +1,110 @@
...
+
+The recommended use cases of Chat2Query are as follows:
+
+- Use the AI capacity of Chat2Query to help you generate complex SQL queries instantly.
+- Test out the MySQL compatibility of TiDB quickly.
+- Explore TiDB SQL features easily.
+
+## Limitation
+
+SQL queries generated by the AI in Chat2Query are not 100% accurate and might still need your further tweak.
```
Comment (qiancai) [inline tidb-cloud/explore-data-with-chat2query.md:26]:
```suggestion
SQL queries generated by the AI are not 100% accurate and might still need your further tweak.
```

**Review by Oreoxmt — APPROVED** [review]
LGTM

File: tidb-cloud/connect-to-tidb-cluster.md [hunk 35]
```diff
@@ -17,9 +17,11 @@ For Serverless Tier clusters, you can connect to your cluster via standard conne
 
     Serverless Tier only [supports TLS connections](/tidb-cloud/secure-connections-to-serverless-tier-clusters.md), which ensures the security of data transmission from your applications to TiDB clusters.
 
-- Chat2Query (Beta)
+- [Chat2Query (beta)](/tidb-cloud/explore-data-with-chat2query.md)
```
Comment (qiancai) [inline tidb-cloud/connect-to-tidb-cluster.md:20]:
```suggestion
- Connect via [Chat2Query (beta)](/tidb-cloud/explore-data-with-chat2query.md)
```

**Pull request merged**